src/module/system/damage/formula.ts

```diff
@@ -79,15 +79,18 @@ function createDamageFormula(
     // Hold onto base terms for matching reasons (such as adding extra base dice)
     const baseTerms = [...typeMap.values()].flat();
 
-    // Dice always stack
+    // Sometimes a weapon may add base damage as bonus modifiers or dice. We need to auto-generate these
+    const BONUS_BASE_LABELS = ["PF2E.ConditionTypePersistent"].map((l) => game.i18n.localize(l));
```

Comment:
Does this really need to be run through a `map`?

Reply:
I probably did that with the assumption that it might need to be expanded, and doing a map is less code for when that happens.

It doesn't seem like splash has the same issue since we don't add splash to the damage label, so I guess we can cross that bridge later.